internal/telemetry/event.go

```diff
@@ -105,7 +109,12 @@ func withEmpty(e bool) eventOpt {
 	}
 }
 
-func withCommandPath(cmd *cobra.Command) eventOpt {
+type CmdName interface {
```

Comment:
golangcilint rule - every interface should have comment.
Interfaces should be declared on the top of the file etc.

Reply:
the comment rule is disable by default, if we were to enable it it may be a large backfill 
interfaces at the top  do you know the rule for this?, personally I like declare things as close as possible to where they are needed